packages/utils/src/format.ts

```diff
@@ -9,12 +8,12 @@ export const NEAR_NOMINATION_EXP = 24;
 /**
  * Number of indivisible units in one NEAR. Derived from {@link NEAR_NOMINATION_EXP}.
  */
-export const NEAR_NOMINATION = new BN('10', 10).pow(new BN(NEAR_NOMINATION_EXP, 10));
+export const NEAR_NOMINATION = BigInt(10 ** NEAR_NOMINATION_EXP);
```

Comment:
please check that this handle the right precision, before it was `Bigint(10)**BigInt(EXP)`

Reply:
yes, should be `BigInt(10)**BigInt(EXP)` but we need to change the `target` option to `es2016` or later to make it work. Are you ok with that?

Reply:
yes, since

`BigInt(10**24) == 999999999999999983222784n`

while

`BigInt(10)**BigInt(24) == 1000000000000000000000000n`